### PR TITLE
Improve lit tests: added missed parameters to make its more robust and reliable

### PR DESCRIPTION
**Pull request by andreytr4intel**

### CODE REVIEW

**Comment by andreytr4intel** [comment]
Recently I've developed a set of scripts (https://github.com/KhronosGroup/SPIRV-LIT-Converter) for converting LIT tests from SPIRV-LLVM-Translator format to LLVM-SPIRV-Backend (https://github.com/KhronosGroup/LLVM-SPIRV-Backend) format. During developing of these scripts, I've found several issues (missed parameters) from original source SPIRV-LLVM-Translator's lit tests, that can be corrected, in my opinion, to improve lit tests, make it more robust and reliable.
I suggest my fixes to include to original repository. Hope, it would be helpful.

**Comment by andreytr4intel** [comment]
@AlexeySachkov , @MrSidims , @svenvh , @bader , can you, please, take a look at this, or add someone as reviewers?

File: test/transcoding/OpConstantSampler.ll [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
 ; CHECK-SPIRV: ConstantSampler {{[0-9]+}} [[SamplerID0:[0-9]+]] 3 1 0
 ; CHECK-SPIRV: ConstantSampler {{[0-9]+}} [[SamplerID1:[0-9]+]] 0 0 0
-; CHECK-SPIRV: SampledImage {{.*}} [[SamplerID0]]
-; CHECK-SPIRV: SampledImage {{.*}} [[SamplerID1]]
+; CHECK-SPIRV: SampledImage {{.*}} {{.*}} {{.*}} [[SamplerID0]]
```
Comment (bader) [inline test/transcoding/OpConstantSampler.ll:23]: `{{.*}} -> {{[0-9]+}}`
Please, apply to all other instances of `{{.*}}` next to each other.

**Review by bader — CHANGES_REQUESTED** [review]

**Review by bader — APPROVED** [review]

File: test/transcoding/OpConstantSampler.ll (hunk 1 above)
Comment (andreytr4intel) [reply test/transcoding/OpConstantSampler.ll]: Fixed, checked.

**Review by MrSidims — APPROVED** [review]

**Comment by andreytr4intel** [comment]
Thanks!

**Pull request merged**